File: candidates/back/Dbo/Education.cs
using System;
using System.Collections.Generic;

namespace AiCandidate.Dbo
{
    public partial class Education
    {
        public ulong Id { get; set; }
        public ulong IdCandidate { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string Level { get; set; }

        public virtual Candidate IdCandidateNavigation { get; set; }
    }
}

File: offers/front/src/app/models/education.ts
import { ApiEducation } from './api/api-education';

export class Education {
    public readonly id: bigint;
    public readonly title: string;
    public readonly description: string;
    public readonly level: string;

    constructor(json: ApiEducation) {
        this.id = json.id;
        this.title = json.title;
        this.description = json.description;
        this.level = json.level;
    }
}